webui/react/src/pages/WorkspaceList/WorkspaceCard.tsx

```diff
@@ -32,14 +32,15 @@ const WorkspaceCard: React.FC<Props> = ({ workspace, fetchWorkspaces }: Props) =
   });
   const loadableUser = useObservable(userStore.getUser(workspace.userId));
   const user = Loadable.getOrElse(undefined, loadableUser);
-
+  const testId = `card-${workspace.name}`
+  
   return (
     <Card
       actionMenu={!workspace.immutable ? menu : undefined}
       size="small"
       onClick={(e: AnyMouseEvent) => handlePath(e, { path: paths.workspaceDetails(workspace.id) })}
       onDropdown={onClick}>
-      <div className={workspace.archived ? css.archived : ''}>
+      <div className={workspace.archived ? css.archived : ''} data-testid={testId}>
```

Comment:
Is this OK? Am I opening up a HTML/JS injection attack here?

Reply:
i guess we could test with a workspace that had `>` in it's name

Reply:
tested and seems good, it gets quoted `data-testid="card-dnj > <script>alert(&quot;Hello!&quot;)</script>"`